pygluu/kubernetes/gui_config.py

```diff
@@ -0,0 +1,18 @@
+class Config(object):
+    DEBUG = False
+    TESTING = False
+    SECRET_KEY = "supersecret"
+    UPLOAD_FOLDER = "/"
+    ALLOWED_EXTENSIONS = {"crt", "yaml"}
+
+
+class DevelopmentConfig(Config):
+    DEBUG = True
+
+
+class ProductionConfig(Config):
+    SECRET_KEY = ""
```

Comment:
Assuming `ProductionConfig` is loaded when using `FLASK_ENV=production`, there should be a way to populate `SECRET_KEY` as it's needed by CSRF, session, cookies, and possibly other Flask extensions.

Reply:
@iromli here some updates, i've restructured the project folder for gui installer 
- production config will be loaded by default, and to enable debug mode you can do `SECRET_KEY=**** pygluu-kubernetes-gui --debug=true`
- `SECRET_KEY` can be filled by set envvars e.g `SECRET_KEY="secret" pygluu-kubernetes-gui` but this method is not quite handy, do you have any suggestion?